libff/algebra/curves/bls12_377/bls12_377.sage

```diff
@@ -209,14 +263,15 @@ def g2_fast_subgroup_check_coefficients(g2_order, q, r, t1, h2):
     """
     # For P in E'(Fq2), ψ satisfies:
     #   [q1]P = [t1]ψ(P) - ψ^2(P)
-    # where ψ is the untwist-frobenius-twist endomorphism
+    # where ψ is the untwist-frobenius-twist endomorphism defined on
+    # E'(Fq2) (i.e. the twist used to build a compact representation of G2)
```

Comment:
```suggestion
    # E'(Fq2) (i.e. where "twist" is the twist used to build a compact representation of G2)
```

Reply:
I'll resolve this - it can be ignored.
I thought `i.e.` referred to \psi here, but if it refers to `E'(Fq2)` then it makes sense.